.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
- [ ] OPTIONAL
The `.vscode` is appearing in your project structure. It's a good practice to keep all files and folders which are irrelevant to the project development in the `.gitignore` file. Kindly endeavor to the `.vscode` directory, as well as other _less important files_ in the `.gitignore` list.